avionics/tests/SampleTest.cpp

```diff
@@ -0,0 +1,17 @@
+#include "gtest/gtest.h"
+#include "iostream"
+/*
+ * Sample Test Template
+ * 
+ * Add Tests: 
+ *      For adding new tests, you will also need to link those tests in CMakeLists.txt.
+ *
+ * Run Tests:
+ *      Build project in build folder, then run "make test"
+ */
+
+TEST(SampleTest, SampleTestTemplate)
+{
```

Comment:
I think I deleted the sample test some while back once we actually had actual tests to use as samples; no need to keep it I think.